conn/physic/units.go

```diff
@@ -195,7 +235,7 @@ func (e ElectricResistance) String() string {
 // Set sets the ElectricResistance to the value represented by s. Units are to
 // be provided in "Ohm", "Ohms" or "Ω" with an optional SI prefix: "p", "n",
 // "u", "µ", "m", "k", "M", "G" or "T".
-func (f *ElectricResistance) Set(s string) error {
+func (r *ElectricResistance) Set(s string) error {
```

Comment:
String() uses `e` above. Please use the same one letter for every methods of a type. I failed to notice that in the previous change.